### PR TITLE
Code Review: Amend the License Owners

### PR DESCRIPTION
**Pull request by Coder206**
This PR is mostly just changing the names of the owners of the project's files to "Team Janitor" as discussed with @jankeromnes in IRC.

### CODE REVIEW

File: lib/streams.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ const log = require('./log');
 const objectStreams = new Map();
 
 // Get `object[key]` as a stream (may still be receiving new data).
-
```
Comment (Coder206) [inline lib/streams.js]: I want to keep things consistent with the other files

File: lib/streams.js [hunk 2]
```diff
@@ -28,7 +27,6 @@ exports.get = function (object, key) {
 };
 
 // Read a stream into `object[key]`.
-
```
Comment (Coder206) [inline lib/streams.js]: See previous

File: lib/streams.js [hunk 3]
```diff
@@ -61,7 +59,6 @@ exports.set = function (object, key, readable) {
 };
 
 // Remove any stream affected to `object[key]`.
-
```
Comment (Coder206) [inline lib/streams.js]: See previous

File: static/js/blog.js [hunk 4]
```diff
@@ -1,3 +1,6 @@
+// Copyright © Team Janitor. All rights reserved.
+// The following code is covered by the AGPL-3.0 license.
+
```
Comment (Coder206) [inline static/js/blog.js]: This should be under the license as well because it seems to be ours.

File: static/js/janitor-new.js [hunk 5]
```diff
@@ -50,7 +50,7 @@ Array.forEach($$('.tabs'), function (element) {
 Array.forEach(document.querySelectorAll('form[method]'), function (form) {
   setupAsyncForm(form);
   form.addEventListener('submit', function (event) {
-    // set form to pending status
+    // Set form to pending status
```
Comment (Coder206) [inline static/js/janitor-new.js]: I want to keep the comments consistent

**Comment by jankeromnes** [comment]
> In other news, when looking over the code, I noticed a few occurrences similar to this: https://github.com/JanitorTechnology/janitor/blob/master/lib/machines.js#L410
Could we have these sort of statements at the top of the files like so:
https://github.com/JanitorTechnology/janitor/blob/master/lib/machines.js#L12
Or is there location strategic to get up-to-date data?

Thanks for noticing this! (Tip: When you click on a line to get these links, you can also type the letter `Y` on your keyboard to get commit-specific URLs that remain stable over time.)

I think it's OK to use `db.get('thing')` both locally (e.g. when you just need it in that function), and at the top of a file (e.g. when you use `thing` in many places in that file).

But in `lib/machines.js` for example, moving `const ports = db.get('ports')` to the top of the file would conflict with:

https://github.com/JanitorTechnology/janitor/blob/f9dc2d88e2645d923307db59c9023631a989dfe1/lib/machines.js#L195

This can of course be addressed by renaming it one of the conflicting symbols. TL;DR feel free to hoist `db.get('thing')`s to the top of files, but please make sure to fix any conflicts in the process. 😄

**Comment by Coder206** [comment]
@jankeromnes Thank you for the review!

> (Tip: When you click on a line to get these links, you can also type the letter Y on your keyboard to get commit-specific URLs that remain stable over time.)
Neat, thanks! The URLs are https://github.com/JanitorTechnology/janitor/blob/f9dc2d88e2645d923307db59c9023631a989dfe1/lib/machines.js#L410 and https://github.com/JanitorTechnology/janitor/blob/f9dc2d88e2645d923307db59c9023631a989dfe1/lib/machines.js#L12 in that case :)

> > In other news, when looking over the code, I noticed a few occurrences similar to this: https://github.com/JanitorTechnology/janitor/blob/master/lib/machines.js#L410
> > Could we have these sort of statements at the top of the files like so:
> > https://github.com/JanitorTechnology/janitor/blob/master/lib/machines.js#L12
> > Or is there location strategic to get up-to-date data?
> 
> Thanks for noticing this! (Tip: When you click on a line to get these links, you can also type the letter `Y` on your keyboard to get commit-specific URLs that remain stable over time.)
> 
> I think it's OK to use `db.get('thing')` both locally (e.g. when you just need it in that function), and at the top of a file (e.g. when you use `thing` in many places in that file).
> 
> But in `lib/machines.js` for example, moving `const ports = db.get('ports')` to the top of the file would conflict with:
> 
> [janitor/lib/machines.js](https://github.com/JanitorTechnology/janitor/blob/f9dc2d88e2645d923307db59c9023631a989dfe1/lib/machines.js#L195)
> 
> Line 195 in [f9dc2d8](/JanitorTechnology/janitor/commit/f9dc2d88e2645d923307db59c9023631a989dfe1)
>  const ports = {}; 
> 
> This can of course be addressed by renaming it one of the conflicting symbols. TL;DR feel free to hoist `db.get('thing')`s to the top of files, but please make sure to fix any conflicts in the process. smile

Good catch, I missed that. Thanks! :)


> Approved, many thanks for addressing this!
> 
> I think the `All rights reserved.` doesn't actually mean anything, but it also doesn't hurt to have it, so +1 .
> 
> > Submodule dockerfiles updated 19 files
> 
> Did you also sneak in a Dockerfiles submodule upgrade? :) That's cool with me too.

Thank you! I think the Dockerfiles submodule upgrade was an artefact of using `git pull` for this keeping this PR in sync with master.

**Pull request merged**